contracts/src/rollup/ValidatorWalletCreator.sol

```diff
@@ -29,15 +30,14 @@ contract ValidatorWalletCreator is Ownable {
         emit TemplateUpdated();
     }
 
-    function createWallet() external returns (address) {
+    function createWallet(address _executor, address _owner) external returns (address) {
         ProxyAdmin admin = new ProxyAdmin();
         address proxy = address(
             new TransparentUpgradeableProxy(address(template), address(admin), "")
         );
-        admin.transferOwnership(msg.sender);
-        ValidatorWallet(proxy).initialize();
-        ValidatorWallet(proxy).transferOwnership(msg.sender);
-        emit WalletCreated(proxy, msg.sender, address(admin));
+        admin.transferOwnership(_owner);
+        ValidatorWallet(payable(proxy)).initialize(_executor, _owner);
+        emit WalletCreated(proxy, _executor, _owner, address(admin));
```

Comment:
As-is, this breaks an assumption of the validator software that there will only ever be one validator wallet for its address. We might want to either include the creator as an indexed field in the event, or make executor and owner both start out as `msg.sender`.